.travis.yml

```diff
@@ -1,26 +1,50 @@
+sudo: false
+
 language: python
 
 python:
   - 2.7
   - 3.3
   - 3.4
+  - 3.5
 
 env:
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
-  - DJANGO="Django>=1.8,<1.9"
+  - DJANGO=1.5
+  - DJANGO=1.6
```

Comment:
We have some projects still running on Django 1.1 :fearful: Not sure where the border is